src/components/CalloutBanner.tsx

```diff
@@ -44,6 +44,7 @@ const CalloutBanner = ({
             src={image}
             alt={alt}
             w={imageWidth}
+            maxW={maxImageWidth}
```

Comment:
this applies to images only? Chakra also uses the `w` prop

Reply:
Correct. We need to use `width` because that is a Next Image prop and that will let it know how to resize the image. `w` is just a Chakra prop, a style prop.